package.json

```diff
@@ -19,7 +19,9 @@
   ],
   "dependencies": {
     "@atomist/automation-client": "^0.6.6",
-    "app-root-path": "^2.0.1"
+    "app-root-path": "^2.0.1",
+    "axios-mock-adapter": "^1.13.1",
```

Comment:
axios-mock-adapter should probably be in devDependencies.

Reply:
makes sense